independent-publisher-connectors/Electricity Maps/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1239 @@
+                            "type": "number",
+                            "description": "Hydro power production forecast",
+                            "x-ms-summary": "Hydro"
+                          },
+                          "nuclear": {
+                            "type": "number",
+                            "description": "Nuclear power production forecast",
+                            "x-ms-summary": "Nuclear"
+                          },
+                          "oil": {
+                            "type": "number",
+                            "description": "Oil power production forecast",
+                            "x-ms-summary": "Oil"
+                          },
+                          "solar": {
+                            "type": "number",
+                            "description": "Solar power production forecast",
+                            "x-ms-summary": "Solar"
+                          },
+                          "unknown": {
+                            "type": "number",
+                            "description": "Unknown power production forecast",
+                            "x-ms-summary": "Unknown"
+                          },
+                          "wind": {
+                            "type": "number",
+                            "description": "Wind power production forecast",
+                            "x-ms-summary": "Wind"
+                          }
+                        },
+                        "type": "object",
+                        "description": "Power Production Breakdown data",
+                        "x-ms-summary": "Power Production Breakdown"
+                      },
+                      "powerProductionTotal": {
+                        "type": "number",
+                        "description": "Power Production Total Forecast",
+                        "x-ms-summary": "Total Production"
+                      }
+                    },
+                    "type": "object"
+                  },
+                  "type": "array",
+                  "description": "Power Production forecast data",
+                  "x-ms-summary": "Power Production Forecast"
+                },
+                "updatedAt": {
+                  "type": "string",
+                  "description": "Date and time when forecast has been updated",
+                  "x-ms-summary": "Updated At"
+                },
+                "zone": {
+                  "type": "string",
+                  "description": "Country or Zone",
+                  "x-ms-summary": "Zone"
+                }
+              },
+              "type": "object"
+            }
+          }
+        },
+        "operationId": "BreakdownForecast",
+        "summary": "Get Power Production Forecast",
+        "description": "Retrieves the forecasted power production breakdown of an area by production type. It can either be queried by zone identifier or by geolocation"
```

Comment:
@VitaliiSorokin all operations descriptions should end with a period, No period required only if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)